test/bundler-integration/token-paymaster/btpm-undeployed-wallet.ts

```diff
@@ -266,6 +290,13 @@ describe("Biconomy Token Paymaster (with Bundler)", function () {
         "nonce"
       );
 
+      const signatureWithModuleAddress = ethers.utils.defaultAbiCoder.encode(
```

Comment:
You could in theory use `makeEcdsaModuleUserOpWithPaymaster` from userOp.ts, which could make things little bit shorter